atlasdb-cli/versions.lock

```diff
@@ -456,7 +450,7 @@
             ]
         },
         "com.palantir.tritium:tritium-api": {
-            "locked": "0.6.0",
+            "locked": "0.6.0-beta",
```

Comment:
yep. we know this will break compass, but it's easily fixed on their side.

Do we want to hold up tonight's release on this, though?